eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -218,6 +218,8 @@ jobs:
 
     ### Signing variables
     - ${{ if eq(parameters.sign, 'True') }}:
+      - name: _SignDiagnosticFilesArgs
+        value: ''
```

Comment:
Where is that variable used?

Reply:
This is produced in `install-diagnostic-certificates.yml` and then passed to the build command.